hw/ip/otbn/doc/dv/index.md

```diff
@@ -155,75 +167,107 @@ This paragraph implies eight coverage points (four flags times two values) for t
 
 #### ADD
 
+This instruction uses the `R` encoding schema, with covergroup `enc_r_cg`.
+
 - Cross the three possible signs (negative, zero, positive) for each input operand (giving 9 points).
 
 #### ADDI
 
-As for `ADD`.
+This instruction uses the `I` encoding schema, with covergroup `enc_i_cg`.
+
+Specific points as for `ADD`.
 
 #### LUI
 
-Nothing beyond immediate toggle coverage.
+This instruction uses the `U` encoding schema, with covergroup `enc_u_cg`.
 
 #### SUB
 
+This instruction uses the `R` encoding schema, with covergroup `enc_r_cg`.
+
 - Cross the three possible signs (negative, zero, positive) for each input operand (giving 9 points).
 
 #### SLL
 
+This instruction uses the `R` encoding schema, with covergroup `enc_r_cg`.
+
 - A shift of a nonzero value by zero.
 - A shift of a value by `0x1f` which leaves the top bit set.
```

Comment:
If the bottom bit of the source is set, so you want to set the bottom bit both ways.

Reply:
I don't understand this message. Could you elaborate further? I definitely *think* I'm going to collect coverage on operand values (although not in this patch).